### PR TITLE
Add a bytecode listing

### PR DESCRIPTION
**Pull request by rwy7**
The bytecodes listed do not reflect reality. They are more of a proposal
of what our bytecodes will be, when b9 is completed.

Signed-off-by: Robert Young <rwy0717@gmail.com>

### CODE REVIEW

File: docs/docs/Operators.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+(pop-into-local  index)  ;; ( value -- )
+```
+
+### Arguments
+```lisp
+(push-from-argument index)  ;; ( -- value )
+(push-from-argument-i)      ;; ( index -- value )
+(pop-to-argument index)     ;; ( value -- )
+(push-arguments)            ;; ( -- arguments )
+(push-rest)                 ;; ( -- rest )
```
Comment (dnakamura) [inline docs/docs/Operators.md]: what is rest? I'm guessing its anything past your declared arguments

File: docs/docs/Operators.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+(enter-environment nslots)          ;; ( -- )
+(leave-environment)                 ;; ( -- )
+```
+
+The environment register is used to share state across function scopes. The environment is a chain 
+
+### This Register
+```lisp
+(call-with-this)  ;; ( args* callee this -- result )
+(push-this)       ;; ( )
```
Comment (dnakamura) [inline docs/docs/Operators.md]: I assume you mean
`(push-this)       ;;( -- this)`

File: docs/docs/Operators.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+## Extension: Exception handling
+see: https://github.com/WebAssembly/exception-handling/blob/master/proposals/Level-1.md
+
+## Extension: Atomics?
+see: https://llvm.org/docs/Atomics.html
+
+## Extension: Inline caches
+see: https://jandemooij.nl/blog/2017/01/25/cacheir/
+
+# Values in b9
```
Comment (dnakamura) [inline docs/docs/Operators.md]: Maybe want to move this section to the top of the document?

File: docs/docs/Operators.md (hunk 1 above)
Comment (rwy7) [reply docs/docs/Operators.md]: Yep. My current idea is that, `push-arguments` and `push-rest` will copy args from the stack, and into an array object.

File: docs/docs/Operators.md (hunk 2 above)
Comment (rwy7) [reply docs/docs/Operators.md]: derp, yes.

File: docs/docs/Operators.md (hunk 3 above)
Comment (rwy7) [reply docs/docs/Operators.md]: Sure, will do.

**Pull request closed without merging**